CMakeLists.txt

```diff
@@ -15,6 +15,10 @@ option (ENABLE_PIC
    "Enables building of position independent code for static library components."
    ON
 )
+option (ENABLE_BUILD_FOR_PPA "Maintainer-only option for preparing PPA build" OFF)
+if (ENABLE_SHARED_BSON AND ENABLE_BUILD_FOR_PPA)
```

Comment:
In refreshing myself on static/shared linking from prior conversations. From [this prior conversation](https://github.com/mongodb/libmongocrypt/pull/111/files#r497556530), a consumer linking to a shared libmongocrypt does not need to link to libbson.

I understand that to mean that `libmongocrypt.pc` does not need to specify `libbson-static-1.0` under `Requires`. But `libmongocrypt-static.pc` does need to specify `libbson-static-1.0` under `Requires`.

Is that correct? If so, why does this add `PPA` as a special case?

Reply:
OK, thank you for the explanation. I had forgotten that `libbson-dev` was not part of the PPA. I still think we can remove the `Requires: libbson-static-1.0` from `libmongocrypt.pc` (but not `libmongocrypt-static.pc`. But I think that can be out of scope of this ticket.